vcr/patch.py

```diff
@@ -43,14 +44,16 @@
 
 # Try to save the original types for requests
 try:
```

Comment:
Do you still need this try/except block?

Reply:
@pquentin I believe so, at least vcrpy can be installed without any requests around:

```console
# cd "$(mktemp -d)"
# python3.8 -m venv venv
# source venv/bin/activate
# pip install vcrpy
[..]
# pip freeze | grep requests
[no output]
```

What do you think?

Reply:
Good point, thanks.